styles.css

```diff
@@ -322,8 +323,11 @@ a:focus-visible {
     }
     .settings-input__check-wrapper {
       width: 100%;
-
       display: flex;
+      gap: 10px;
+    }
+
+    .settings-input__check-wrapper.check-reverse {
```

Comment:
It increases specificity in CSS. Could you try to avoid it? In my codebase, it shouldn't be so hard with the HTML file with just 100 lines of code.
Why did you add `gap` in here? What's the point? Is it possible to avoid of using row-reverse? It doesn't look necessary in the code you've implemented.